src/getHumanAge.test.js

```diff
@@ -1,10 +1,22 @@
 'use strict';
 
-describe('getHumanAge', () => {
-  const { getHumanAge } = require('./getHumanAge');
+const { getHumanAge } = require('./getHumanAge');
 
+describe('getHumanAge', () => {
   test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+    expect(getHumanAge).toBeInstanceOf(Function);
+  });
+});
+
+describe('convertToHuman', () => {
+  test('should convert cat and dog ages to human years correctly', () => {
```

Comment:
The test description should mention that it's testing the 'getHumanAge' function.